src/4d.js

```diff
@@ -37,23 +37,20 @@ module.exports = function(hljs) {
     className: 'number',
     begin: '(-?)(\\b0[xX][a-fA-F0-9]+|(\\b[\\d]+(\\.[\\d]*)?|\\.[\\d]+)([eE][-+]?[\\d]+)?)(?!D)'
   };
-
+  //Does not support unicode characters (should be replaced by \p{L} when possible)
```

Comment:
Does JS not support \p{L} yet?

Reply:
https://caniuse.com/#feat=mdn-javascript_builtins_regexp_property_escapes

Ah looks like mostly firefox being an issue?